src/ui/components/SearchInput/index.js

```diff
@@ -42,6 +42,9 @@ export default class SearchInput extends React.Component {
       // There is no transitionstart event, but animation will start if there is no value.
       this.setState({ animating: true });
     }
+    // Handle the possibility that onTransitionEnd will never fire if
+    // the CSS animation is a no-op due to the size of text in the label.
+    window.setTimeout(this.onTransitionEnd, 250);
```

Comment:
It doesn't look like it jumps but would need to see it on a real device to know for sure.